Teensy/src/gpio/gpio.cpp

```diff
@@ -11,59 +11,59 @@
 #include <Arduino.h>
 #include "gpio.h"
 
-static int32_t gpio_getWheelSpeed();
-static uint32_t gpio_getEngineRevs();
-static int32_t gpio_getDamperPosition();
-static int32_t gpio_getGearPosition();
-static uint32_t gpio_getSteeringWheelPosition();
-static int32_t gpio_getStrain();
+static daq_WheelSpeed_t gpio_getWheelSpeed();
```

Comment:
Needs it's own datatype to store all 4 wheel speeds

Reply:
Changed :)